src/perf.ts

```diff
@@ -55,6 +55,7 @@ export async function launch(options: Options) {
 		let timedOut = false;
 		let promise: Promise<string | undefined>;
 		const abortController = new AbortController();
+		process.on('SIGINT', () => abortController.abort());
```

Comment:
Not sure if it's a concern in this context, but you may want to make sure to remove this listener later on, since the present of any listener for SIGINT will prevent the default exit behavior (i.e. even after this function returns)